engine/classes/Elgg/Context.php

```diff
@@ -51,7 +45,7 @@ public function peek() {
 	 * @return void
 	 */
 	public function push($context) {
-		array_push($this->stack, $context);
+		array_push($this->stack, "$context");
```

Comment:
BTW, Just noticed this on php.net: "If you use array_push() to add one element to the array it's better to use ```$array[] = ``` because in that way there is no overhead of calling a function."